### PR TITLE
feat: Audit Logs Docs

### PR DESCRIPTION
**Pull request by AnaghHegde**

### CODE REVIEW

File: .github/drafts/15938/auditLog-first-draft.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+1. Users - Select the specific user to filter activities based on the user from the dropdown list
+2. Events - Filter events based on events. 
+3. ResourceId - Select the resource-id from the json to filter the 
+4. Date Range - Filter data by specific date range you wish to filter
+
+**Audit Log Properties:**
+
+
+<p id="gdcalert1" ><span style="color: red; font-weight: bold">>>>>>  gd2md-html alert: inline image link here (to images/image1.png). Store image on your image server and adjust path/filename/extension if necessary. </span><br>(<a href="#">Back to top</a>)(<a href="#gdcalert2">Next alert</a>)<br><span style="color: red; font-weight: bold">>>>>> </span></p>
```
Comment (IAmAnubhavSaini) [inline .github/drafts/15938/auditLog-first-draft.md:30]: Image is not reachable; and doesn't get rendered.

File: .github/drafts/15938/auditLog-first-draft.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+
+!<p style="color: red; font-weight: bold">screenshot of the auditLog json</p><a href="#gdcalert1">alert1</a>
```
Comment (IAmAnubhavSaini) [inline .github/drafts/15938/auditLog-first-draft.md:33]: The image is missing here.

File: .github/drafts/15938/auditLog-first-draft.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
+**Intro**
```
Comment (IAmAnubhavSaini) [inline .github/drafts/15938/auditLog-first-draft.md:1]: Do we not do headings via `#`?

File: .github/drafts/15938/auditLog-first-draft.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+**Filtering audit logs **
+
+
+
+1. Users - Select the specific user to filter activities based on the user from the dropdown list
+2. Events - Filter events based on events. 
+3. ResourceId - Select the resource-id from the json to filter the 
+4. Date Range - Filter data by specific date range you wish to filter
+
```
Comment (vuiets) [inline .github/drafts/15938/auditLog-first-draft.md:26]: @jnikhila @Pranay105 not sure who to tag from the user education pod for this but we'll also have to talk about each event and when is it recorded here

We have about 25+ events that we log to start with, here's the list
https://www.notion.so/appsmith/579671b3503b48b1a5967b969e3823e2?v=92a738b1dbc5438c97ca27ca87bc30fd

**Comment by vuiets** [comment]
@jnikhila @Pranay105 just reiterating what I added in one of the review comments here.

We log about 25+ different events. This is missing from the draft but the event names and descriptions are available here. Hope this helps.
https://www.notion.so/appsmith/579671b3503b48b1a5967b969e3823e2?v=92a738b1dbc5438c97ca27ca87bc30fd

File: .github/drafts/15938/auditLog-first-draft.md (hunk 2 above)
Comment (AnaghHegde) [reply .github/drafts/15938/auditLog-first-draft.md:33]: I could not upload it. I tried that but docs team will add this when they work on it.

File: .github/drafts/15938/auditLog-first-draft.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+ 
+
+**Filtering audit logs **
+
+
+
+1. Users - Select the specific user to filter activities based on the user from the dropdown list
+2. Events - Filter events based on events. 
+3. ResourceId - Select the resource-id from the json to filter the
```
[the remark this answers is not in the document]
Comment (IAmAnubhavSaini) [reply .github/drafts/15938/auditLog-first-draft.md]: A resource represents the modified entity e.g. page, application, query etc. ResourceId is the id that uniquely identifies that entity in database.

[the remark this answers is not in the document]
Comment (vuiets) [reply .github/drafts/15938/auditLog-first-draft.md]: Thanks @IAmAnubhavSaini! @AnaghHegde rephrasing a bit here
> Resource ID - A resource represents the modified entity be it a application, page, datasource, query, JSObject or workspace. Resource ID uniquely identifies every entity you create in your Appsmith instance. Here it helps you filter the logs for specific entity.

**Review by vuiets — APPROVED** [review]

**Pull request merged**